src/WebJobs.Script.WebHost/Controllers/HostController.cs

```diff
@@ -216,7 +216,6 @@ public async Task<IActionResult> SetState([FromBody] string state)
         [HttpGet]
         [HttpPost]
         [Authorize(AuthenticationSchemes = AuthLevelAuthenticationDefaults.AuthenticationScheme)]
-        [RequiresRunningHost]
```

Comment:
In v2 we changed the route for this - it is no longer an admin route, so it doesn't need RequiresRunningHost anymore, since the HostAvailabilityCheckMiddleware runs for all non admin routes and ensures the host is running. See v1 route here: https://github.com/Azure/azure-functions-host/blob/v1.x/src/WebJobs.Script.WebHost/Controllers/AdminController.cs#L205.

Reply:
I thought we had changed it back, as this requires us to reserve another route under a different base URI

Reply:
I think this was changed long ago